ethereum_bridge/Cargo.toml

```diff
@@ -26,6 +26,8 @@ abciplus = [
     "namada_proof_of_stake/abciplus",
 ]
 
+testing = []
```

Comment:
Added a `testing` feature to `namada_ethereum_bridge` crate for guarding test helper functionality (this is not a great way of doing it, but follows the pattern we're currently using - a dedicated test utils crate would be better)